mem/mem_test.mbt

```diff
@@ -0,0 +1,94 @@
+// Copyright 2024 International Digital Economy Academy
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+fn test_load(m : Loadable) -> Result[Unit, String] {
+  inspect(m.load_int(1, Big), content="33752069")?
+  inspect(m.load_int64(1, Big), content="144964032628459529")?
+  inspect(m.load_int(1, Little), content="84148994")?
+  inspect(m.load_int64(1, Little), content="650777868590383874")?
+  Ok(())
+}
+
+test "load bytes" {
+  test_load(Bytes::[1, 2, 3, 4, 5, 6, 7, 8, 9])?
+}
+
+test "load bytes view" {
+  test_load(
+    {
+      Bytes::[0, 1, 2, 3, 4, 5, 6, 7, 8, 9][1..]
+    },
```

Comment:
`moon fmt` added unnecessary `{}` here.

Reply:
cc @Yoorkin

Reply:
Fixed. The fix will apply in the next release.